CP04OSSM/CP04OSSM_D00014_ingest.csv

```diff
@@ -0,0 +1,32 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/cpm3/syslog/cpm_status*.txt,CP04OSSM-MFC31-00-CPMENG000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl36/syslog/*.syslog.log,CP04OSSM-MFD35-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl36/adcp/*.adcp.log,CP04OSSM-MFD35-01-ADCPSJ000,telemetered,Available,
+mi.dataset.driver.presf_abc.dcl.presf_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl36/presf/*.presf.log,CP04OSSM-MFD35-02-PRESFC000,telemetered,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl36/velpt2/*.velpt2.log,CP04OSSM-MFD35-04-VELPTB000,telemetered,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl36/pco2w/*.pco2w.log,CP04OSSM-MFD35-05-PCO2WB000,telemetered,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl36/phsen2/*.phsen2.log,CP04OSSM-MFD35-06-PHSEND000,telemetered,Available,
+#mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl37/syslog/*.syslog.log,CP04OSSM-MFD37-00-DCLENG000,telemetered,Not Expected,DCL failed on deployment
+#mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl37/optaa2_decimated/*.optaa2.log,CP04OSSM-MFD37-01-OPTAAD000,telemetered,Not Deployed,Instrument not deployed
+#mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl37/ctdbp2/*.ctdbp2.log,CP04OSSM-MFD37-03-CTDBPE000,telemetered,Not Expected,DCL failed on deployment
+#mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl37/dosta2/*.dosta2.log,CP04OSSM-MFD37-04-DOSTAD000,telemetered,Not Expected,DCL failed on deployment
+#mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CP04OSSM/D00014/cg_data/dcl37/zplsc/*.zplsc.log,CP04OSSM-MFD37-07-ZPLSCC000,telemetered,Not Expected,DCL failed on deployment

```

Comment:
lines commented out for DCL37 failure look good; OPTAA removed from MFN & commented out - good.